tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/ExpressionEvaluationTest.java

```diff
@@ -139,7 +139,8 @@ public void variableReferenceEvaluationTest() throws BallerinaTestException {
         // error variable test
         debugTestRunner.assertExpression(context, ERROR_VAR, "SimpleErrorType", "error");
         // anonymous function variable test
-        debugTestRunner.assertExpression(context, ANON_FUNCTION_VAR, "function (string,string) returns (string)",
+        debugTestRunner.assertExpression(context, ANON_FUNCTION_VAR, "function isolated function (string,string) " +
```

Comment:
Is having function before the function type expected here?

Reply:
This happens because the `stringValue()` method of the `io.ballerina.runtime.internal.values.FPValue` adds a `function` prefix, in https://github.com/ballerina-platform/ballerina-lang/blob/865850e24ff2882178085c7b0ed3948248e5df8c/bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/FPValue.java#L86.

I had a call with Waruna and he will try removing the prefix, which would solve the issue.

Reply:
Sorry, missed this earlier. But I think this is intended. For values where the spec doesn't define a string representation, the implementation defines its own and we seem to have used the basic type-descriptor + type as the string representation.

For example, https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TypedescValueImpl.java#L93. So I guess we have to keep this?

Reply:
+1. I have reverted the change. @NipunaRanasinghe I think as per @MaryamZi comment, we need to fix it in the debugger side.

Reply:
@warunalakshitha @MaryamZi Ack.. I'll modify the debugger side to preprocess the type string before sending it to the frontend.

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/30604